packages/terra-form-input/src/terra-dev-site/test/form-input/UnsupportedTypesInputField.test.jsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import classNames from 'classnames/bind';
+import InputField from '../../../InputField';
+import styles from './common/Input.test.module.scss';
+
+const cx = classNames.bind(styles);
+
+const unsupportedTypes = ['date', 'datetime-local', 'month', 'time', 'week', 'color', 'file', 'range', 'checkbox', 'radio', 'button', 'reset', 'submit'];
+
+class UnsupportedTypesInputField extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      isInvalid: false,
+      isIncomplete: false,
+      required: false,
+    };
+
+    this.toggleInvalid = this.toggleInvalid.bind(this);
+    this.toggleIncomplete = this.toggleIncomplete.bind(this);
+  }
+
+  createInputFieldTypeExamples(arr) {
+    const inputFieldTypeExamples = [];
+    for (let i = 0; i < arr.length; i += 1) {
+      inputFieldTypeExamples.push((
+        <div className={cx('content-wrapper')}>
+          <InputField
+            inputId={`unsupported-${arr[i]}-input`}
+            error="Error message."
+            help="Help message."
+            isInvalid={this.state.isInvalid}
+            isIncomplete={this.state.isIncomplete}
+            required={this.state.required}
+            inputAttrs={{
+              name: `html-original-${arr[i]}-input`,
+              type: arr[i],
+              ariaLabel: `Input type=${arr[i]}`,
```

Comment:
Same as comment for `SupportedTypesInputField` Test.

Reply:
updated here https://github.com/cerner/terra-core/pull/3386/commits/451995dda70a0c87b88b1a69e7e802692ae84c2d